contrib/pipelines/secadvisor/README.md

```diff
@@ -1,34 +1,85 @@
 # Security Advisor Pipeline
 
-Implements the IBM Security Advisor pipeline
+The IBM Security Advisor pipeline extracts flow data from skydive, filters out
+external traffic, transforms flow records by adding status information, and
+flow classification (egress, ingress, internal), batches flows records together
+into streams (by time or count), then encodes these blocks in JSON form, and
+compresses using GZIP and finally stores as objects into any S3 compatible
+target.
 
+## Deploy Skydive
 
-## Setup Skydive
-
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
 
 ```
 make
-cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+sudo $(which skydive) allineone -c etc/skydive.yml.default
 ```
 
 ## Setup Minio
```

Comment:
You can the title be 'deploy' (align to 'deploy skydive')

Reply:
ok